### ISSUE TITLE
Model worse after refinement of labels

### ISSUE THREAD

**Issue by rshteyn** [DeepLabCut/DeepLabCut#1393]
Hello,

I refined the labels of my first model, and created and trained a new dataset. However, when I evaluated the new model, it looks much worse than the first model.

For the new model, the train error is 1.42 pixels, and the test error is 73.66 pixels -- compared to my first model which had train error of 1.52 pixels, Test error of 3.85 pixels. Why did the test error get much higher, when I assumed it would decrease?
<p>
Before I retrained the model, I made sure to edit the pose_cfg.yaml file by changing:

**init_weights: /gsfs0/public/deeplabcut/2.0-p3.68/envs/tfgpu/lib/python3.6/site-packages/deeplabcut/pose_estimation_tensorflow/models/pretrained/resnet_v1_50.ckpt**
<p>
to this line:

**init_weights: /gsfs0/data/shteynr/FMS-shteynr-2021-05-26/dlc-models/iteration-0/FMSMay26-trainset95shuffle1/train/snapshot-1030000**

I also made sure to update the shuffle to 1 in the code that I ran:

**dlc.train_network(config_path, gputouse=2, shuffle=1, max_snapshots_to_keep=7)
dlc.evaluate_network(config_path,Shuffles=[1], plotting=True)**

Is it possible that the two datasets did not merge properly?

**Comment by MMathisLab** [comment]
@rshteyn welcome; can you fill out the template please? In general we ask that Q's go to the forum, bugs go here. 

 train error is 1.42 pixels, and the test error is 73.66 pixels -- compared to my first model which had train error of 1.52 pixels, Test error of 3.85 pixels

> so, this looks like overfitting - how much more did you train the model, given it was already at 1M iterations? I.e., the train went down (seems very low, most humans are not less that 2 pixel accurate), and the test is very bad as the model is over-trained.



In the workflow, if the model is not performing well you extract outlier frames, these get clustered, and you re-label example bad frames. then you must merge. This changes the iteration #, and you create a new training dataset. Then you train from scratch again, i.e. don't edit pose_cfg.yaml, and then train.

**Comment by rshteyn** [comment]
Sorry about the template mistake.

I didn't realize that the model could be overtrained through this workflow. So as I understand it, this overtraining occurred due to me editing the pose_cfg.yaml file. The new merged training dataset, iteration 1, contains the merged set of frames, so editing the pose_cfg.yaml file caused the frames from iteration 0 to be trained twice rather than once along with iteration 1. Should I try reseting init_weights for iteration 1 back to how they were, and retraining?
